configs/base.eslintrc.json

```diff
@@ -19,5 +19,5 @@
     "browser": true,
     "es2022": true
   },
-  "ignorePatterns": ["node_modules", "lib", "*.d.ts", ".eslintrc.js"]
+  "ignorePatterns": ["node_modules", "lib", "*.d.ts", ".eslintrc.*js"]
```

Comment:
You could now revert even this change ...

Reply:
No needed to rename the .eslintrc.js to .eslintrc.cjs...

Reply:
Ah of course you have to rename then, sorry.